[CTOR-1546]-Refresh applications-mulesoft-restapi  doc page (#4294)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/applications-mulesoft-restapi.md

A service account with proper access in the right Mulesoft environment/organization has to be used.

> This account has to own the privileges to manage Applications, Servers, Clusters and Messages Queues on the designated environment.

Reviewer: ```suggestion
This account must have sufficient privileges to manage Applications, Servers, Clusters and MessagesQueues in the designated environment.
```
